src/arrayMethodReduce.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, initialValue = 0) {
+    let i = 1;
+    let accumulator = initialValue;
+
+    arguments.length === 1
```

Comment:
1. for one variable - one ternary operator
2. firstly you need to write the variable you define and then it's value or condition